docs/sources/reference/api.md

```diff
@@ -868,6 +870,28 @@ It is an approximation with the following caveats:
 These make it generally more helpful for larger queries.
 It can be used for better understanding the throughput requirements and data topology for a list of matchers over a period of time.
 
+## Volume
+
+The `/loki/api/v1/index/volume` and `/loki/api/v1/index/volume_range` endpoints can be used to query the index for volume information about label and label-value combinations. This is helpful in exploring the logs Loki has ingested to find high or low volume streams. The `volume` endpoint returns aggregations over the entire time period, returning a Prometheus style vector response with one timestamp and datapoint per matching set of labels or series. The `volume_range` endoint returns a Prometheus style matrix response, with multiple timestamps and corresponding datapoints per matching set of labels or series. The number of timestamps returned when querying `volume_range` will be determined by the provided `step` parameter.
```

Comment:
I think it would be useful to adopt the same language as the `query` and `query_range` endpoint docs where they talk about `a point in time` vs `over a range of time`